luacheck-scm-1.rockspec

```diff
@@ -0,0 +1,82 @@
+package = "luacheck"
+version = "scm-1"
+source = {
+   url = "git+https://github.com/mpeterv/luacheck.git"
```

Comment:
Why not our fork?

Reply:
I have not known that we have already forked it

Fixed

Reply:
Did you check it after the change?

Reply:
Revert to upstream. Our fork should be rebased on upstream